src/main/java/util/csvparsing/CsvMonumentConverter.java

```diff
@@ -29,7 +29,7 @@ public static Monument convertCsvRowToMonument(String csvRow) {
             // I also assumed that if any columns were empty, they would be null
             switch (columnIndex) {
                 case 0: // Submitted By
-                    result.setSubmittedBy(value);
+                    //result.setSubmittedBy(value);
```

Comment:
Planning to fix this class under the other task that is currently assigned to me.